README.md

```diff
@@ -1,13 +1,20 @@
 # Tyk Operator
 
-[Tyk Gateway](https://github.com/TykTechnologies/tyk) is a modern, ultra-performant, purpose-built, open source API Gateway. 
- 
-Tyk Operator brings Full Lifecycle API Management capabilities to Kubernetes. Configure Ingress, APIs, Security Policies, Authentication, 
- Authorization, Mediation and more - all using GitOps best practices with Custom Resources and Kubernetes Native primitives.
+[Tyk Gateway](https://github.com/TykTechnologies/tyk) is a modern, ultra-performant, purpose-built, open source API
+Gateway.
+
+Tyk Operator brings Full Lifecycle API Management capabilities to Kubernetes.
+Configure Ingress, APIs, Security Policies, Authentication, Authorization, Mediation and more - all using GitOps best
+practices with Custom Resources and Kubernetes-native primitives.
 
 ---
 
-[Documentation](#documentation) | [Learning with Videos](#learning-with-videos) | [Quickstart Examples](#quickstart-examples) | [IDE Integration](#ide-integration) | [Community](#community)
+- [Documentation](#documentation)
+- [Learning with Videos](#learning-with-videos)
+- [Quickstart Examples](#quickstart-examples)
+- [IDE Integration](#ide-integration)
+  - [VS Code](#vs-code)
+- [Community](#community)
```

Comment:
nit: I am a fan of the intro being piped - rather than a list. Keeps it clean & succinct. If we can remove the `VS Code` subnav also?

Reply:
af36d13 should resolve